src/io/flutter/FlutterMessages.java

```diff
@@ -17,6 +18,7 @@
 
 public class FlutterMessages {
   public static final String FLUTTER_NOTIFICATION_GROUP_ID = "Flutter Messages";
+  public static final String BAZEL_CONFIG_WARN_PREFIX = "Configuration warning: ";
```

Comment:
Matching on this string could have false positives. Fyi @jwren.
We should consider making the string for bazel configuration warnings more unique and we need to document that this prefix needs to match the bazel tools.
An alternative to this design would be to display all bazel warnings in the g3 dart plugin. That would also allow showing the warning even before you run an app.

Reply:
Is there a way to read configuration from flutter.json from within g3, or have flutter.json return a variable from elsewhere in g3? 

Using the g3 dart plugin would be nice too. This maybe wouldn't need to be specific to actions like run/test and would just detect whether the setup is running on mac without remote Dart. 

For now I think I'll just add a todo item. I wanted a message to appear for tools run on the command line as well.

Reply:
lets file a bug along with the todo.